spec/models/like_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'Validation tests for Likes' do
+    subject { Comment.new(author_id: 2, post_id: 2) }
+
+    before { subject.save }
+
+    it 'Find author_id to be an integer' do
+      subject.author_id = 'ismail'
+      expect(subject).to_not be_valid
+    end
+
+    it 'Find author_id to be an integer greater or equal to zero' do
+      subject.author_id = -2
+      expect(subject).to_not be_valid
+    end
+
+    it 'Find post_id to be an integer' do
+      subject.post_id = 'ismail'
+      expect(subject).to_not be_valid
+    end
+
+    it 'Find post_id to be an integer greater or equal to zero' do
+      subject.post_id = -2
+      expect(subject).to_not be_valid
+    end
+  end
+end
```

Comment:
- [ ] After add rspec in the gem file and run the rspec it is giving error. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/179369916-c2fda963-8bb2-4e0d-84b9-4633b9b1f7b1.png)